hardhat.config.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const { argv } = yargs(hideBin(process.argv));
+
+require('@nomiclabs/hardhat-waffle');
+require('@nomiclabs/hardhat-ethers');
+require('@nomiclabs/hardhat-etherscan');
+
+require('solidity-coverage');
+require('hardhat-contract-sizer');
+
+// Load Config
+const configPath = path.join(__dirname, '/config.json');
+const config = fs.existsSync(configPath) ? JSON.parse(fs.readFileSync(configPath, 'utf8')) : {};
+
+const loadAPIKey = (apiKeyName) => {
+    return config.apiKeys ? (config.apiKeys[apiKeyName] ? config.apiKeys[apiKeyName] : '') : '';
+};
+
+// Config
+const configNetworks = config.networks || {};
+const APIKeyEtherscan = loadAPIKey('etherscan');
+
+module.exports = {
+    // Network Config
+    networks: {
+        // Hardhat network
+        hardhat: {
+            gasPrice: 20000000000,
+            gas: 9500000,
+            accounts: [
```

Comment:
Because there is some tests that requires huge amount of ethers. I didn't want to change the test so I just did that. Very ugly I agree. - Seems like the test that were requiring a lot of ethers are not there anymore. Will try to remove them and see what happens

Reply:
In ganache, there was an option to only set the default ETH balance, without configuring the actual accounts. Can you check if it's the case as well?